web/themes/custom/atrium/templates/node/node--srl-howto-instruction--full.html.twig

```diff
@@ -35,13 +35,80 @@
     </div>
   {% endif %}
 
-  {% set right_column = '' %}
-  {% if content.field_blocks|render or content.field_forms|render %}
-    {% set right_column %}
+  {% set right_column %}
+    <div class="mobile:display-none jcc-share--desktop">
+      {% include '@molecules/blocks/share/share.twig' with {
+        social_links: {
+          items: [
+            {
+              id: "print",
+              title: "Print",
+              src: "/libraries/courtyard/public/images/graphics/print.svg",
+              label: "Print current page",
+              renderer: 'patternlab',
+            },
+            {
+              id: "email",
+              title: "Email",
+              src: "/libraries/courtyard/public/images/graphics/email.svg",
+              label: "Email page to yourself",
+              uses_modal: true,
+              modal_content: drupal_entity('webform', 'mark_my_place'),
+              renderer: 'patternlab',
+            },
+            {
+              id: "text",
+              title: "Text",
+              src: "/libraries/courtyard/public/images/graphics/text.svg",
+              label: "Email page to yourself",
+              uses_modal: true,
+              modal_content: drupal_entity('webform', 'mark_my_place'),
+              renderer: 'patternlab',
+            },
+          ]
+        }
+      } only %}
+    </div>
+
+    {% if content.field_blocks|render or content.field_forms|render %}
       {{ content.field_blocks }}
       {{ content.field_forms }}
-    {% endset %}
-  {% endif %}
+    {% endif %}
+  {% endset %}
+
+  <div class="desktop:display-none jcc-share--mobile">
+    {% include '@molecules/blocks/share/share.twig' with {
```

Comment:
@robdecker From our experiences upgrading courts.ca.gov with responsive support, physical relocating elements based on media was not reliable with CSS when dealing with exact positioning on a page. Did you have something CSS in mind that I may have overlooked?

The other option is to use JavaScript possibly but I don't recall why we did not do so previously. JS might be worth the try if you agree.

BTW... Drupal does not handle confirm/error message displays when duplicating webforms on the same page. I am going to need to create unique webforms since the other confirmation behaviors will either not work (such as a modal which does not solve inline error suggestions) or not preferred (e.g. redirecting to a different page).

Reply:
Ah... discovered grid... https://stackoverflow.com/questions/23238673/moving-elements-in-dom-for-responsive-web-design.

Not certain it will work since the right column could include other elements besides sharing but let me give it a go.

Reply:
Another suggestion from @robdecker is use twig to solve media-based placement.

Reply:
Trying absolute positioning with a spacer element to create the real needed when appearing in right column. Grid CSS will be difficult to swap position if relevant sidebar content blocks are used (e.g. relevant forms and notes).

There's already an overlap issue already with callout block used in the body content that can impact any right column use.